### PR TITLE
Convert tabs to spaces in css/ directories with few lint errors

### PR DESCRIPTION
**Pull request by foolip**
This follows the same approach as in https://github.com/web-platform-tests/wpt/pull/19824
to identify directories in css/ with few lint errors and removes the
lint.whitelist exceptions for those.

For css/css-counter-styles/ some newlines were also added/removed to
arrive at a reasonable style.

### CODE REVIEW

File: lint.whitelist [hunk 1]
```diff
@@ -398,31 +398,22 @@ NON-EXISTENT-REF: css/CSS2/normal-flow/blocks-017.xht
...
 INDENT TABS: css/css-backgrounds/*
 INDENT TABS: css/css-color/*
 INDENT TABS: css/css-conditional/*
-INDENT TABS: css/css-counter-styles/*
 INDENT TABS: css/css-flexbox/*
 INDENT TABS: css/css-fonts/*
-INDENT TABS: css/css-grid/*
-INDENT TABS: css/css-images/*
-INDENT TABS: css/css-lists/*
+INDENT TABS: css/css-images/support/1x1-green.svg
```
Comment (astearns) [inline lint.whitelist]: was this addition intended?

File: css/css-counter-styles/ethiopic-numeric/css3-counter-styles-068.html [hunk 2]
```diff
@@ -18,18 +18,19 @@
 <p class="instructions">Test passes if the two columns are the same, ignoring the suffix.</p>
 
 
-<div class='test'><ol>
```
Comment (astearns) [inline css/css-counter-styles/ethiopic-numeric/css3-counter-styles-068.html]: was this change intended?

File: css/css-counter-styles/ethiopic-numeric/css3-counter-styles-069.html [hunk 3]
```diff
@@ -18,85 +18,77 @@
 <p class="instructions">Test passes if the two columns are the same, ignoring the suffix.</p>
 
 
-<div class='test'><ol start='10'>
```
Comment (astearns) [inline css/css-counter-styles/ethiopic-numeric/css3-counter-styles-069.html]: was this change intended?

**Review by astearns — APPROVED** [review]
LGTM aside from my non-blocking questions.

File: lint.whitelist (hunk 1 above)
Comment (foolip) [reply lint.whitelist]: Yes, that file is weird ("Generator: Adobe Illustrator", `transform="matrix(1 0 0 1.375 0.0015 -0.313)" `, plenty of unneeded markup) and I didn't want to deal with it.

File: css/css-counter-styles/ethiopic-numeric/css3-counter-styles-068.html (hunk 2 above)
Comment (foolip) [reply css/css-counter-styles/ethiopic-numeric/css3-counter-styles-068.html]: Yes, the style of the tests in this directory was so weird I couldn't make sense of it without adding a few newlines.

File: css/css-counter-styles/ethiopic-numeric/css3-counter-styles-069.html (hunk 3 above)
Comment (foolip) [reply css/css-counter-styles/ethiopic-numeric/css3-counter-styles-069.html]: Yep.

**Comment by foolip** [comment]
Added this to the description:
> For css/css-counter-styles/ some newlines were also added/removed to
> arrive at a reasonable style.

**Comment by foolip** [comment]
Looks like I'm still failing lint in css/css-lists/, will have to push a fix for that in a bit.

**Comment by foolip** [comment]
@astearns would you mind also reviewing the new changes in css/css-counter-styles/? They're all whitespace only.

**Comment by foolip** [comment]
wpt-firefox-nightly-stability has failed on Taskcluster because css/css-transitions/properties-value-001.html was flaky, but that's not plausibly caused by these changes. The test was run because it includes the modified vendorPrefix.js.

**Comment by foolip** [comment]
Querying for `is:different` for [Chrome](https://wpt.fyi/results/css?diff&filter=ADC&run_id=313700012&run_id=323020011), [Firefox](https://wpt.fyi/results/css?diff&filter=ADC&run_id=319130011&run_id=319120010) and [Safari](https://wpt.fyi/results/css?diff&filter=ADC&run_id=346120010&run_id=323040007) confirms no differences in results.

**Comment by astearns** [comment]
> @astearns would you mind also reviewing the new changes in css/css-counter-styles/? They're all whitespace only.

Seems fine to me.

**Comment by emilio** [comment]
Looks fine but out of curiosity, why the choice of four spaces in some tests but two in others?

**Comment by foolip** [comment]
@emilio for files that were entirely indented with tabs and not a mixed mess, I looked at other tests in the same directory and those named similarly for what indentation level was more common. In other words, I tried to make it fit in as well as possible.

**Comment by emilio** [comment]
Sounds good :)

**Comment by r12a** [comment]
@foolip can you confirm that the changes were all formatting related and there were no substantive code changes?  If there were substantive changes, i'd need you to point them out so that i can consider applying them to the tests in the i18n test suite from whence these came.

**Comment by foolip** [comment]
@r12a I can confirm that, the only changes that weren't merely replaced tabs with some number of spaces were in css/css-counter-styles/, where the only additional change was addition and removal of newline characters. In other words, all whitespace.
